src/Catel.MVVM/Catel.MVVM.Shared/MVVM/Commands/Command.cs

```diff
@@ -244,7 +244,7 @@ public void Execute(object parameter)
         public void Execute(TExecuteParameter parameter)
         {
 #pragma warning disable 4014
-            ExecuteAsync(parameter, false);
+            ExecuteAsync(parameter, false).Wait();
```

Comment:
As I have found you must await the async method so the exception is carried with.
We cannot use 'await' since the method must be declared as async. So you can only .Wait().